app/src/main/java/com/kamron/pogoiv/pokeflycomponents/ocrhelper/OcrHelper.java

```diff
@@ -1130,15 +1132,18 @@ public ScanData scanPokemon(@NonNull GoIVSettings settings,
         }
         Optional<Integer> evolutionCost = getPokemonEvolutionCostFromImg(pokemonImage,
                 ScanArea.calibratedFromSettings(POKEMON_EVOLUTION_COST_AREA, settings));
-        Pair<String, String> moveset = getMovesetFromImg(pokemonImage,
-                estimatedLevelRange,
-                ScanArea.calibratedFromSettings(POKEMON_POWER_UP_CANDY_COST, settings),
-                ScanArea.calibratedFromSettings(POKEMON_EVOLUTION_COST_AREA, settings));
-        String moveFast = null;
-        String moveCharge = null;
-        if (moveset != null) {
-            moveFast = moveset.first;
-            moveCharge = moveset.second;
+
+        String moveFast = "";
```

Comment:
If the moveset scan fails `moveFast` and `moveCharge` should be null, otherwise `ScanData` / `ScanResult` data structures would behave like the moveset scan was successful. Please initialize this variables to null instead of empty String